drivers/dma/dma_dw_common.c

```diff
@@ -263,8 +263,10 @@ int dw_dma_config(const struct device *dev, uint32_t channel,
 		LOG_DBG("dest data size: lli_desc %p, ctrl_lo %x", lli_desc, lli_desc->ctrl_lo);
 
 		lli_desc->ctrl_lo |= DW_CTLL_SRC_MSIZE(msize) |
-			DW_CTLL_DST_MSIZE(msize) |
-			DW_CTLL_INT_EN; /* enable interrupt */
+			DW_CTLL_DST_MSIZE(msize);
+
+		if (cfg->dma_callback)
```

Comment:
needs brackets to make CI happy and get a merge

Reply:
_All checks have passed
1 skipped and 22 successful checks_

Should I still add brackets?

Reply:
Yes, you should. No idea why checkpatch did not catch this.

Reply:
fixed now